src/HttpServer.ts

```diff
@@ -65,31 +65,30 @@ export class HttpServer {
     }
 
     // TODO: use this when changing to jsonrpc
-    async rootHandler(req: any, res: any): Promise<void> {
+    async rootHandler(req: Request, res: Response): Promise<void> {
```

Comment:
😍

Reply:
however, no other property of `req` is accessed or called so I'd destructure it here to `{body: requestBody}` to make it clearer.